src/types/index.ts

```diff
@@ -191,6 +193,42 @@ export interface StarlightClient<
    */
   model<S extends keyof D>(slug: S): DynamicModelInstance<D[S]>
 
+  /**
+   * Returns a {@link DynamicFormSelector}, which is a {@link FormSelector}
+   * with support for creating {@apilink FormInstance | FormInstances} using the dynamic syntax.
```

Comment:
Same as before, we can rephrase this:

```suggestion
   * Returns a {@link DynamicFormSelector}, which provides support for 
   * creating {@apilink FormInstance | FormInstances} using the dynamic syntax.
```